### PR TITLE
Finished car dealership project

### PR DESCRIPTION
**Pull request by a-dozen-beans**
Finished assignment, all tests passing

Any feedback on the code quality and what could be done better or more effectively is more than welcome and much appreciated :) @santiagobasulto @jasonmeeks @martinzugnoni

### CODE REVIEW

**Comment by santiagobasulto** [comment]
This PR will be reviewed by: @kellim

File: dealership/vehicles.py [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+    def purchase_price(self):
+        return self.sale_price() - self.purchase_multiplier * self.miles
 
 
 class Car(Vehicle):
-    pass
+    sale_multiplier = 1.2
+    purchase_multiplier = 0.004
+    interest_multiplier = 1.07
+    lease_multiplier = 1.2
```
Comment (kellim) [inline dealership/vehicles.py:24]: I like the way you have made these class attributes - this seems cleaner than the way I did it.

File: dealership/contracts.py [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+
+        total = sale_price + \
+            (sale_price * lease_multiplier / self.length_in_months)
+
+        if self.customer.is_employee():
+            return total * 0.9
+        return total
+
+    def monthly_value(self):
+        return self.total_value() / self.length_in_months
```
Comment (kellim) [inline dealership/contracts.py:44]: Your code is neatly formatted, and you do a good job of making sure the files meet PEP-8 guidelines.

**Review by kellim — COMMENTED** [review]
Your code looks good to me - I can't see anything wrong with it. I wish I had more feedback to give, but you thought of better ways to do things than I did. I think you have done a great job.

**Pull request closed without merging**